server/monitor/src/main/java/org/apache/accumulo/monitor/util/ParameterValidator.java

```diff
@@ -22,6 +22,8 @@
 public interface ParameterValidator {
 
   String ALPHA_NUM_REGEX = "\\w+";
+  // Allow the special default table IDs
+  String ALPHA_NUM_REGEX_TABLE_ID = "[\\!\\+]?(\\w)+";
```

Comment:
I don't think you need to escape either the exclamation or the plus sign in the character class. Also, you should not put the `\\w` in a capture group (remove parens).

Reply:
I agree "[!+]?\\w+" should work just fine.

Reply:
Yup, thanks.  Fixed in 19ba632